dts/arm/ti/am64x_am243x_mcu.dtsi

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2025 Texas Instruments Incorporated
+ *
```

Comment:
This file make sense. We should update existing AM64 boards to this file to after this PR got  merged.

Reply:
noted, will have to remember this

Reply:
it's up to you. I'm not a fan of adding additional tasks to a pull-request because it increases the duration. Since no one has approved yet, it's still possible. If you have some time, would be great to fix it now.

Reply:
edited phyboard_electra DT in a new commit

Reply:
Thanks a lot!